js/pages/cohort-definitions/components/cohortfeatures/tableConfig/FormattingOptions.js

```diff
@@ -1,4 +1,6 @@
 define(function (require, exports) {
+	
+	var numeral = require('numeral');
```

Comment:
Do we still need `cohortfeatures` under cohort definitions? Why not to drop that code completely?

Reply:
Would be a good thing to do in a separate PR.

Reply:
@anthonysena , I'd propose then to open the appropriate issue and merge the current PR after that

Reply:
Done in #1898